package.json

```diff
@@ -31,23 +31,19 @@
     "safe-buffer": "~5.1.1"
   },
   "devDependencies": {
-    "@types/node": "^8.0.47",
     "airtap": "^0.0.5",
     "faucet": "^0.0.1",
     "istanbul": "^0.4.2",
     "istanbul-coveralls": "^1.0.3",
     "rimraf": "^2.6.2",
     "standard": "^11.0.0",
-    "tape": "^4.8.0",
-    "ts-node": "^4.1.0",
-    "typescript": "^2.6.2"
+    "tape": "^4.8.0"
   },
   "browser": {
     "./immediate.js": "./immediate-browser.js"
   },
   "scripts": {
-    "test": "standard && node test.js --stderr | faucet && npm run test-ts",
-    "test-ts": "ts-node --no-cache test.js | faucet",
+    "test": "standard && node test.js --stderr | faucet",
```

Comment:
Come to think of it, what does `--stderr` argument imply here?

Reply:
I have no idea.. Wondered that myself

Reply:
=> #155